packages/vuetify/src/composables/nested/nested.ts

```diff
@@ -217,8 +217,7 @@ export const useNested = (props: NestedProps) => {
 export const useNestedItem = (id: Ref<unknown>, isGroup: boolean) => {
   const parent = inject(VNestedSymbol, emptyNested)
 
-  const uidSymbol = Symbol(getUid())
-  const computedId = computed(() => id.value ?? uidSymbol)
+  const computedId = computed(() => id.value)
```

Comment:
@KaelWD If both `undefined` and `null` are legit prop value/id, do you have any concern of not using Symbol here?

Reply:
getUid returns a number and I'm also not sure how this is related

Reply:
This impacts ListItem's `isSelected` state when `item-value` is null or undefined